jkube-kit/jkube-kit-quarkus/src/test/java/org/eclipse/jkube/quarkus/QuarkusUtilsTest.java

```diff
@@ -283,6 +285,43 @@ public void resolveQuarkusLivenessPath_withLivenessPathSet_shouldReturnValidPath
     assertThat(resolvedHealthPath).isNotEmpty().isEqualTo("liveness");
   }
 
+  @Test
+  public void resolveQuarkusStartupPath_withStartupPathSet_shouldReturnValidPath() {
+    // Given
+    Properties properties = new Properties();
+    properties.setProperty("quarkus.smallrye-health.startup-path", "startup");
+    javaProject.setProperties(properties);
+    // When
+    String resolvedStartupPath = resolveQuarkusStartupPath(javaProject);
+    // Then
+    assertThat(resolvedStartupPath).isNotEmpty()
+            .isEqualTo("startup");
+  }
+
+  @Test
+  public void isStartupEndpointSupported_withQuakursVersionBefore2_1_shouldReturnFalse() {
+    // Given
+    javaProject.setDependencies(quarkusDependencyWithVersion("2.0.3.Final"));
+
+    // When
+    boolean result = isStartupEndpointSupported(javaProject);
+
+    // Then
+    assertThat(result).isFalse();
+  }
+
+  @Test
+  public void isStartupEndpointSupported_withQuakursVersionAfter2_1_shouldReturnFalse() {
+    // Given
+    javaProject.setDependencies(quarkusDependencyWithVersion("2.9.2.Final"));
+
+    // When
+    boolean result = isStartupEndpointSupported(javaProject);
+
+    // Then
+    assertThat(result).isTrue();
```

Comment:
Maybe you should refactor test name from `isStartupEndpointSupported_withQuakursVersionAfter2_1_shouldReturnFalse` to 
`isStartupEndpointSupported_withQuakursVersionAfter2_1_shouldReturnTrue` because you're asserting opposite of name.

Reply:
You're right. I'll update it.